Lib/email/utils.py

```diff
@@ -215,6 +215,14 @@ def parsedate_to_datetime(data):
 
 
 def parseaddr(addr):
+    """
+    Parse address into its constituent 
+    realname and email address parts
```

Comment:
This should be a single line ending with a period.  Instead of 'address', use the variable name from the prototype: 'addr'.